### DISCUSSION TITLE
How can i train a model don't use pre trained model?

### DISCUSSION THREAD

**Discussion by morestart** [ultralytics/yolov5#5181] [Q&A] [answered]
i use this command `--cfg yolov5custom.yaml  --weights ''`
but i get this error:
`AssertionError: '' acceptable suffix is ['.pt']`

**Comment by morestart** [comment, marked as answer]
i use yolov5 v6.0, my current solution is disable `check_suffix(weights, '.pt')`

**Reply by glenn-jocher** [reply]
@morestart thanks for letting us know about this. Another user reported the same issue on Windows. I'm working on a fix for this now.

**Reply by glenn-jocher** [reply]
@morestart good news 😃! Your original issue may now be fixed ✅ in PR #5192. This PR allows empty strings to pass through `check_suffix()` without error, which multiple windows users had reported was causing issues.

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Reply by morestart** [reply]
best work! thanks for your reply~

**Reply by glenn-jocher** [reply]
Great!
